src/Script/RamCalculations.ts

```diff
@@ -161,7 +161,7 @@ function parseOnlyRamCalculate(
 
   // Finally, walk the reference map and generate a ram cost. The initial set of keys to scan
   // are those that start with the name of the main script.
-  let ram = RamCostConstants.Base;
+  let ram: number = RamCostConstants.Base;
```

Comment:
What changed such that this was necessary? I know that "as const" allows for tighter inferences, but AFAIK it should be the same as initializing this to 1.6, which would infer the type number.

Reply:
When we use `as const`, the type is narrowed down as tight as possible. For example:
```typescript
let ram = RamCostConstants.Base;
```
In this case, the type of `ram` is not `number`, it's `1.6`. We cannot assign any other values. You can check it at https://github.com/bitburner-official/bitburner-src/actions/runs/10408238673/job/28825245929.

Reply:
Hmm, I guess it's acting like `let ram = 1.6 as const`, i.e. the const becomes transitive. Makes sense I suppose, but not what I was expecting.